src/Symfony/Bridge/PhpUnit/DeprecationErrorHandler.php

```diff
@@ -178,17 +178,38 @@ public static function register($mode = false)
         }
     }
 
+    /**
+     * Returns true if STDOUT is defined and supports colorization.
+     *
+     * Reference: Composer\XdebugHandler\Process::supportsColor
+     * https://github.com/composer/xdebug-handler
+     *
+     * @return bool
```

Comment:
this annotation should be removed and replaced by a return-type on the hasColorsSupport (same in other files below)

Reply:
?? Sorry, you've lost me!

Reply:
Oups sorry, branch 2.7 which doesn't support scalar type hints.